### DISCUSSION TITLE
“Actions” in the repository view is wrongly translated into Japanese

### DISCUSSION THREAD

**Discussion by noritada** [community/community#102509] [Mobile] [answered]
In the repository view of the mobile app, "Actions" is translated to "アクションアクション" ("Actions actions" in Japanese).

The issue can be seen only in Japanese; no problem in English.

Japanese:
![image](https://github.com/community/community/assets/20850/d9dab635-8610-41a3-97d6-1927e10eb843)

English:
![image](https://github.com/community/community/assets/20850/258a2df5-0b7b-47af-bdf8-106dc881b446)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, バージョン 1.146.0 (120968236)
Device: iPhone SE 2 (iOS 16.7.2)
</details>

**Comment by aconsuegra** [comment, marked as answer]
Thanks for reporting the issue @noritada! I'll report it immediately and we'll ship a fix as soon as possible.

**Reply by noritada** [reply]
Updating to v1.150.0 resolved this issue. Thanks!

![image](https://github.com/community/community/assets/20850/e9b0846e-3bed-436f-8542-c8874141259d)
